src/client/dfuse/dfuse.h

```diff
@@ -333,7 +335,8 @@ struct fuse_lowlevel_ops *dfuse_get_fuse_ops();
 #define DFUSE_REPLY_ATTR(ie, req, attr)					\
 	do {								\
 		int __rc;						\
-		DFUSE_TRA_DEBUG(ie, "Returning attr mode %#o dir:%d",	\
+		DFUSE_TRA_DEBUG(ie, "Returning attr inode %#lx mode %#o dir:%d", \
```

Comment:
(style)  line over 80 characters

Reply:
I've got a fix for this as well which I can include next time but I'll wait for another review to push anything.